src/app/team/page.tsx

```diff
@@ -148,7 +148,7 @@ const HoverComponent = ({ name, post }: Img) => {
 export default function Page() {
 	return (
 		<>
-			<main className="flex flex-col justify-center items-center px-8 graph-paper">
+			<main className="flex flex-col justify-center bg-[#fbfbf2] items-center px-8 ">
```

Comment:
This color looks good, 
but overall the Hero page looks deserted. 
try increasing font size.